### PR TITLE
Core/Packet: WorldState

### PR DESCRIPTION
**Pull request by IntelligentQuantum**
## Changes Proposed:
-  Class Base Packet
-  

## Issues Addressed:

- Closes 

## SOURCE:

Tc
## Tests Performed:

- 
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: src/server/game/Server/Packets/WorldStatePackets.h [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ */
+
+#ifndef WorldStatePackets_h__
+#define WorldStatePackets_h__
+
+#include "Packet.h"
+
+namespace WorldPackets
+{
+    namespace WorldState
```
Comment (Winfidonarleyan) [inline src/server/game/Server/Packets/WorldStatePackets.h:25]:
```cpp
namespace WorldPackets::WorldState
```

Comment (IntelligentQuantum) [reply src/server/game/Server/Packets/WorldStatePackets.h:25]: > 

https://github.com/azerothcore/azerothcore-wotlk/pull/9411/files#diff-27bddf24ddfcf4ede4b55a0a794fada5fd58db1d31bb9694ca851f993f73c949R72

Comment (Winfidonarleyan) [reply src/server/game/Server/Packets/WorldStatePackets.h:25]: Okay

**Review by Winfidonarleyan — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
@IntelligentQuantum @Winfidonarleyan what's the status of this PR?  does it need further testing/review? is it ready to be merged?

don't forget to update the workflow labels please

**Review by Winfidonarleyan — APPROVED** [review]

**Comment by claudiodfc** [comment]
Seems to be working. No crashes or freezes so far.

##### Test case
- Queued for Battleground on debug mode
- Waited 2 minutes until it started.

##### Extras
image

**Pull request merged**